pymc_marketing/clv/models/gamma_gamma.py

```diff
@@ -67,8 +68,35 @@ def expected_customer_spend(
                 random_seed=random_seed,
             ).posterior_predictive["mean_spend"]
 
-    def expected_new_customer_spend(self, n=1, random_seed=None):
-        """Predict expected transaction value of new unobserved customers"""
+    def expected_customer_spend(
+        self,
+        customer_id: Union[np.ndarray, pd.Series],
+        mean_transaction_value: Union[np.ndarray, pd.Series],
+        number_transactions: Union[np.ndarray, pd.Series],
+    ) -> xarray.DataArray:
+        """Expected transaction value per customer
+
+        Adapted from: https://github.com/CamDavidsonPilon/lifetimes/blob/aae339c5437ec31717309ba0ec394427e19753c4/lifetimes/fitters/gamma_gamma_fitter.py#L117  # noqa: E501
```

Comment:
I suggest we also add the paper reference as in the `lifetimes` docstrings. Something like
```
Equation (5) from:
        http://www.brucehardie.com/notes/025/
```